libs/dh/shared/feature-flags/src/lib/feature-flags.ts

```diff
@@ -37,8 +37,8 @@ export const DhFeatureFlagsConfig = makeFeatureFlags({
    * Feature flag example:
    *
    * 'example-feature-flag': {
-   *  created: '01-01-2022',
-   *  disabledEnvironments: [DhAppEnvironment.prod],
+   *     created: '01-01-2022',
+   *     disabledEnvironments: [DhAppEnvironment.prod],
    * },
    *
```

Comment:
suggestion: move the code block on top of the variable.

suggestion: change variable's name to start with a lower-case for consistency.